src/main/java/org/prebid/server/bidder/smartadserver/SmartadserverBidder.java

```diff
@@ -107,6 +103,21 @@ private String getUri() {
                 .toString();
     }
 
+    private Site modifySite(Site site, Integer networkId) {
```

Comment:
One more small change, pls, make methods modifySite and modifyPublisher to be static

Reply:
@muuki88 agree?

Reply:
Will do! I must have missed that comment.